improc/simulator.py

```diff
@@ -802,29 +890,582 @@ def make_galaxy_image(
         rotation = np.deg2rad(rotation)
 
         # regular coordinate grid
+        t0 = time.time()
         x0, y0 = np.meshgrid(np.arange(imsize[1]), np.arange(imsize[0]))
+        cls.runtime['meshgrid'] += time.time() - t0
 
         # transform the coordinates using rotation and translation
+        t0 = time.time()
         x = (x0 - center_x) * np.cos(rotation) + (y0 - center_y) * np.sin(rotation)
         y = (y0 - center_y) * np.cos(rotation) - (x0 - center_x) * np.sin(rotation)
         r = np.sqrt(x ** 2 + y ** 2)
+        cls.runtime['transform'] += time.time() - t0
 
         # first make the bulge using a sersic profile
+        t0 = time.time()
         r0 = sersic_scale
-        bulge = np.exp(-7.67 * (r / r0) ** (1 / 4))
+        bulge = np.exp( -7.67 * np.power(r / r0, 1 / 4) )
```

Comment:
Is there any benefit to doing np.power instead of **?  I was under the impression that numpy would do the same thing in either case.

Reply:
I think there is not difference. I was just trying both options to make sure it doesn't change the runtime. It doesn't.